db_actions.py

```diff
@@ -90,39 +92,40 @@ def checked_out_by(self, device_id):
         :return: UserID
         """
         user_info = self.db_fetch(
-            "SELECT CheckedOutBy from Devices WHERE DeviceID = {}".format(int(
-                device_id)))
+            "SELECT CheckedOutBy from Devices WHERE DeviceID = {}".format(
+                int(device_id)))
         logging.debug(
-            "[db_actions][checked_out_by] DeviceID {} checked out by {}".format(
-                device_id, user_info.get("CheckedOutBy")))
+            "[db_actions][checked_out_by] DeviceID {} checked out by {}".
+            format(device_id, user_info.get("CheckedOutBy")))
         return user_info.get("CheckedOutBy")
 
-    def get_device_name(self, port):
+    def get_device_name(self, location, port):
         """
         :param port: USB port
+        :param location: Location
         :return: DeviceName
         """
         try:
             device = self.db_fetch(
-                "SELECT DeviceName from Devices WHERE Port = '{}'".format(
-                    port))
+                "SELECT DeviceName from Devices WHERE Port = '{}' AND Location = '{}'".
+                format(location, port))
```

Comment:
are these variables in the wrong location? The first `{}` is looking for the Port, but `location` is first in the `format()`